pkg/kubernetes/nulecule.js

```diff
@@ -85,6 +87,98 @@ define([
             return promise;
         }
 
+        function check_nulecule_version(image) {
+            var dfd = $.Deferred();
+            var msgl;
+            var versions = {};
+            debug("checking nulecule specversion");
+            var process = cockpit.spawn(["/usr/bin/atomic", "info", image])
+                .fail(function(ex) {
+                    console.warn(ex);
+                    var message = _("Unable to fetch specversion from image.");
```

Comment:
This happens on my system if I type the following:

 * Type: Nulecule Application
 * Nulecule: projectatomic/helloapache
 * Namespace: projectatomic

Shouldn't we handle this case better with a more descriptive message, if people will run into it often?